### PR TITLE
feat(frontend): add search bar to graphical selector [CF-311]

### PR DESCRIPTION
**Pull request by ollibowers**
It zooooooooooooms
image
**TODO: change actual source of data for search bar**

### CODE REVIEW

**Comment by ollibowers** [comment]
```
  _______
 |      _|_
 |     (._.)
 |      /|
 |
/|\

word: _ g _ _

a b c d - f - h i j k
l m n - p q r s t - v
      w x - z
```

**Comment by Peedee2002** [comment]
Wait hold on this looks not shit anymore

File: frontend/src/pages/GraphicalSelector/styles.js [hunk 1]
```diff
@@ -25,4 +26,15 @@ const SidebarWrapper = styled.div`
   flex: 2;
 `;
 
-export default { Wrapper, GraphPlaygroundWrapper, SidebarWrapper };
+const SearchBarWrapper = styled.div`
+  position: absolute;
```
Comment (gtangelo) [inline frontend/src/pages/GraphicalSelector/styles.js]: nitpick: could we prefer using px here?

File: frontend/src/pages/GraphicalSelector/GraphicalSelector.jsx [hunk 2]
```diff
@@ -148,11 +155,21 @@ const GraphicalSelector = () => {
...
+      displayCourseDetails(id);
+    }
+  };
+
   return (
     <PageTemplate>
       <S.Wrapper>
         <S.GraphPlaygroundWrapper ref={ref}>
-          {loading && <Spinner text="Loading graph..." />}
+          {(!courseCodeList) && <Spinner text="Loading graph..." />}
```
Comment (gtangelo) [inline frontend/src/pages/GraphicalSelector/GraphicalSelector.jsx]: could do?
```
{
  !courseCodeList 
? <Spinner text="Loading graph..." /> 
: <S.SearchBarWrapper><NodeSearchBar ... /></S.SearchBarWrapper>
}
```

**Review by gtangelo — APPROVED** [review]
lgtm! just those comments :)

**Pull request merged**